### ISSUE TITLE
Installers can export installations as excel from OTP

### ISSUE THREAD

**Issue by adamhope** [adamhope/testing-github-and-projects#93]
https://trello.com/c/R3NfIN5o/73-installers-can-export-installations-as-excel-from-otp

As an Installer 
I want an excel spreadsheet of all the installations relating to a specific date range
so that I can plan installations and/or import it into a 3rd party system 

**Comments**
It is assumed that the amount of installs that will be exported is small enough to not require specific maximum limitations (TBC)

There are differences between the 2 current exports (retail and road). The new export will combine into a generic export.

The export will consist of the following columns (check To Be Example xlsx):

- Due Date
- Site ID
- Size
- Address
- Location Description
- Previous Advertiser
- Previous Media
- Previous media going to
- Advertiser to install
- Media to install
- Key Number
- Coming from
- Material ETA
- Install Purchase Order

Because this is merging data from both retail and road then for each road, depending on the install type will determine which fields are populated. In the instance that a field is empty e.g. Key Number for a retail media, then the field is to be populated with 'N/A'

Install Purchase Order is TBC, waiting on Finance department to determine where PO is generated so can be blank for this story.

The first line of the spreadsheet should contain the above field titles in bold.

When the file is generated it should automatically display a 'Save File' dialog to force the user to save locally before it can be viewed. The user is to input the name.

**OUT OF SCOPE:**
- Installation work order heading
- Branding 
- Product 
- From 
- Notes
- Media ID 
- Installed date
- Remove & Install boxes
- Area Name 
- Location 
- skin from (same as "coming from")
